### PR TITLE
Define Makefile with golangci-lint

### PR DESCRIPTION
**Pull request by mmorel-35**
As govet and gofmt are now handled by golangci-lint, there is no need to apply them here.

Signed-off-by: Matthieu MOREL <matthieu.morel35@gmail.com>

### CODE REVIEW

**Comment by squeed** [comment]
While you're at it, do you want to also add a lint.sh for people to run manually? Or just add a Makefile? (can be a separate PR)

File: Makefile [hunk 1]
```diff
@@ -0,0 +1 @@
+include mk/lint.mk
```
Comment (squeed) [inline Makefile:1]: one minor bikeshed: I like to be able to do `make lint` - could you add that as a make target :-). Thanks!

**Review by henry118 — APPROVED** [review]
LGTM

**Comment by mmorel-35** [comment]
@squeed ,
Anything to add here ?

**Review by squeed — APPROVED** [review]

**Pull request merged**